padel/index-last-editions.html

```diff
@@ -61,134 +65,43 @@ <h1 class="navbar-brand navbar-brand_"><a href="index.html"><img class="logo" sr
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Elena M.</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Nombre</th>
-                        <th>Equipo</th>
-                        <th>Nº Pista</th>
-                    </tr>
-                    <tr>
-                        <th>Gari</th>
-                        <th rowspan="2">4</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Maxi</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Nombre</th>
-                        <th>Equipo</th>
-                        <th>Nº Pista</th>
-                    </tr>
-                    <tr>
-                        <th>Begoña</th>
-                        <th rowspan="2">5</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>José Luis</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Nombre</th>
-                        <th>Equipo</th>
-                        <th>Nº Pista</th>
-                    </tr>
-                    <tr>
-                        <th>Gabriel</th>
-                        <th rowspan="2">6</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Mari</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Nombre</th>
-                        <th>Equipo</th>
-                        <th>Nº Pista</th>
-                    </tr>
-                    <tr>
-                        <th>Javi</th>
-                        <th rowspan="2">7</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Manuela</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Nombre</th>
-                        <th>Equipo</th>
-                        <th>Nº Pista</th>
-                    </tr>
-                    <tr>
-                        <th>Juan Enrique</th>
-                        <th rowspan="2">8</th>
-                        <th>Por definir</th>
-                    </tr>
-                    <tr>
-                        <th>Fernando</th>
-                        <th>Por definir</th>
-                    </tr>
-                </table>
+                <h2 class="indent">Fotos</h2>
+
+                <!-- <div class="slider"> -->
+                <div class="bxslider">
+                    <div><img src="img/torneo/medallas.jpg" title="Medallas"></div>

```

Comment:
Images have not been uploaded therefor they are not gonna be shown in production. We could do 2 different things here. Either upload them to the project or use ajax which you have already added to get them from an sotorage.